### ISSUE TITLE
Clean JS

### ISSUE THREAD

**Issue by kmonmousseau** [TheNohman/GuidapLuckyNumberTest#2]
Quand c'est possible, je trouve préférable de ne pas mettre du JS en balise script dans le code HTML, mais d'en faire un fichier séparé (en vue de concat / minifier l'ensemble par exemple).

Il y a aussi un fichier JS, que j'imagine généré automatiquement vu le contenu, qui s'avère être totalement inutile.

**Comment by TheNohman** [comment]
Je suis d'accord avec toi pour le fait de séparer le HTML et JavaScript quand c'est possible, j'ai eu un commit foireux sur la version que vous ai envoyés, ce qui explique le fichier "function.js" vide et que mon JavaScript soit dans mon HTML.

Comme c'est la première fois que j'utilise VueJS, je voulais avoir la vision des bindings sur la même page pour travailler.

**Comment by kmonmousseau** [comment]
👍
